pkg/repositories/models/execution.go

```diff
@@ -56,4 +56,10 @@ type Execution struct {
 	// The user responsible for launching this execution.
 	// This is also stored in the spec but promoted as a column for filtering.
 	User string `gorm:"index" valid:"length(0|255)"`
+	// GORM doesn't save the zero value for ints, so we use a pointer for the State field
+	State *int32 `gorm:"default:0"`
```

Comment:
should this be indexed?

Reply:
Added. This would be useful for filtering